content/parcels/_index.md

```diff
@@ -179,10 +181,29 @@ $ curl -H "Authorization: Bearer [token]" \
         "bedrooms": 3,
         "bathrooms": 2
     },
-    "geomArea": 1231.12, // area of the geometry in square feet
-    "communityPlan": {...communityPlan resource...},
-    "zoning": [...zoning resources...],
-    "overlays": [... overlays resources...],
-    "permits": [... permits resource ...]
+   "lotSizeSF" : 1231,   // the lot size in square feet
+   "lotSizeAcre": 0.028, // the lot size in acre
```

Comment:
this will need to change to `lotSizeAcres` once https://github.com/scoutred/app/issues/585 is done.